ci-operator/step-registry/optional-operators/README.md

```diff
@@ -0,0 +1,40 @@
+# Troubleshooting olm deployment failures
+
+This document should help you to analyze where to look in case that olm deployment has failed. It also describes workflows and which one should be your main concern. Logs location is also mentioned as one of the main subjects.
+* [Workflows](#Workflows):
+* [Logs and how to get them](#Logs-and-how-to-get-to-them):
+
+## Workflows:
+You can find all the workflows that are being executed in following yaml files:
+https://github.com/openshift/release/tree/master/ci-operator/config/redhat-openshift-ecosystem/cvp
+
+I will use `4.7` ocp version in this example:
+The workflow you are most interested in is : **optional-operators-cvp-common-aws**.
+
+*Env variables* together with *dependenciy_overrides* are specified before the workflow, these are used during the steps which are mentioned in workflow itself.
+`CVP-COMMON-AWS` workflow does following: installs an optional operator using the input index image, package and channel and executes a common suite of CVP tests to validate the optional operator.
+
+In case you want to know what individual step does, search for them in https://github.com/openshift/release/tree/master/ci-operator/step-registry/optional-operators
+More information about workflows, individual steps and explained content of folders can be found here: 
+https://gitlab.cee.redhat.com/cvp/wiki/-/blob/master/CVP%20Internal/DPTPProwJobs.md
```

Comment:
This is an internal link so it won't be accessible for upstream users

Reply:
Both wiki and confluence is made internal, I suppose we don't want to share such information.

Reply:
Cancelled whole sentence about the confluence.